common/map_test.cpp

```diff
@@ -568,12 +564,9 @@ TYPED_TEST(MapCollisionTest, Basic) {
   for (int i : llvm::seq(1, 256)) {
     SCOPED_TRACE(llvm::formatv("Key: {0}", i).str());
     EXPECT_TRUE(m.Insert(i, i * 100).is_inserted());
-
-    // Immediately do a basic check of all elements to pin down when an
-    // insertion corrupts the rest of the table.
-    ExpectMapElementsAre(m, MakeKeyValues([](int k) { return k * 100; },
-                                          llvm::seq_inclusive(1, i)));
   }
+  ExpectMapElementsAre(
+      m, MakeKeyValues([](int k) { return k * 100; }, llvm::seq(1, 256)));
   EXPECT_FALSE(m.Contains(257));
```

Comment:
This seems a bit redundant given the previous comparison. But if we're going to keep it, checking that `256` is absent would seem better (`seq(1, 256)` excludes its endpoint, so the last key we insert is 255).

Reply:
Quite redundant and yeah, the wrong edge case. I'll just remove it.